examples/telegram-bot/README.md

```diff
@@ -0,0 +1,22 @@
+# Telegram bot example
+
+This example starts a [Telegram](https://telegram.org/) bot on an [ExpressJS](https://expressjs.com/) server.
+
+[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/new?template=https%3A%2F%2Fgithub.com%2Frailwayapp%2Fexamples%2Ftree%2Fmaster%2Fexamples%telegram-bot)
+
+## ✨ Features
+
+- Telegraf (library to interact with the Telegram bot API)
+- Express
+- TypeScript
+
+## 💁‍♀️ How to use
+
+- Install dependencies `yarn`
+- Connect to your Railway project `railway init`
+- Add the bot token as an environment variable `TELEGRAM_BOT_TOKEN`
```

Comment:
Can you add this to your button config? It should be required ye?

![image](https://user-images.githubusercontent.com/5499880/110032889-8bd4bb00-7ced-11eb-9b0e-7509f24d1b36.png)